webpack/plugins.js

```diff
@@ -41,7 +41,6 @@ const prodPlugins = [
   new SplitByPathPlugin([
     { name: 'vendor', path: [path.join(__dirname, '..', 'node_modules/')] },
   ]),
-  new webpack.optimize.OccurenceOrderPlugin(),
   new webpack.optimize.DedupePlugin(),
```

Comment:
Dedupe plugin has been causing problems on a few projects - let's just remove it; it doesn't cut down the bundle size that much anyway.